.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+# Description
+
+Please describe the change you have made.
+
+## Bump
```

Comment:
Even better is to use [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/) and semantic release

Reply:
That would cause the PR description to look strange. Note that we are using the PR descriptions, and not the individual commits to bump.